### PR TITLE
[FIX] #38 Filter out databases if the 'openerp-connector-worker' is tart...

### PR DESCRIPTION
**Pull request by lmignon**
...ed with the '--db-filter' parameter

### CODE REVIEW

**Comment by lmignon** [comment]
Travis is green on our branch... strange.. https://travis-ci.org/acsone/connector/builds/44331693

File: connector/openerp-connector-worker [hunk 1]
```diff
@@ -82,6 +83,9 @@ class WorkerConnector(workers.Worker):
                 db_names = services['db'].exp_list(True)
             else:
                 db_names = []
+        dbfilter = config['dbfilter']
```
Comment (pedrobaeza) [inline connector/openerp-connector-worker:86]: Isn't this going to fail if the key is not present?

**Comment by guewen** [comment]
The build error seems to be some timing error:

```
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
File "/usr/lib/python2.7/threading.py", line 551, in __bootstrap_inner
File "/home/travis/build/OCA/connector/connector/queue/worker.py", line 313, in run
File "/home/travis/build/OCA/connector/connector/queue/worker.py", line 328, in check_alive
File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
File "/home/travis/build/OCA/connector/connector/session.py", line 81, in session
File "/home/travis/build/OCA/connector/connector/session.py", line 157, in close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 161, in wrapper
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 290, in close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 300, in _close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 262, in print_log
<type 'exceptions.TypeError'>: unsupported operand type(s) for +=: 'NoneType' and 'int'
Exception AttributeError: "'NoneType' object has no attribute 'warning'" in <bound method Cursor.__del__ of <openerp.sql_db.Cursor object at 0x5854c10>> ignored
```

**Comment by guewen** [comment]
It should be added here as well: https://github.com/OCA/connector/blob/8.0/connector/queue/worker.py#L265

File: connector/openerp-connector-worker (hunk 1 above)
Comment (lmignon) [inline connector/openerp-connector-worker:86]: @pedrobaeza dbfilter is always present since a default value is declared in the ConfigManager.

**Comment by guewen** [comment]
Thanks!

:+1:

**Comment by lmignon** [comment]
The forward port to 8.0 will follow once this PR will be merged

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**